### DISCUSSION TITLE
distinguish between adv_ind and Scan response manufacture data.

### DISCUSSION THREAD

**Discussion by asdfhhsuidfg** [hbldh/bleak#1365] [Q&A]
I have a application which uses both adv_ind and scan response to advertise custom data. the custom data will be added along with the manufacture specific data. But i have different custom data in adv_ind and scan response.
The command 

manufacturer_data = advertisement_data.manufacturer_data
print(manufacturer_data)

only gives me the manufacture data in scan response data 
any way to access manufacture data in adv_ind?

Thanks in advance

**Comment by dlech** [comment]
Technically, this is possible on Windows but it is not technically possible on Linux or Mac so Bleak does not support it.

Perhaps you could use service data instead of manufacturer-specific data for one of the values?
